vulkan_generator/DEVDOC.md

```diff
@@ -59,6 +59,12 @@ Presubmit can be run from `./kokoro/presubmit/presubmit.sh`. This will require t
 autopep8==1.6.0
 ```
 
+Note: If you do not have a python environment, ensure that autopep8 is in your path. For default Pyhon installation:
+
+```
+export AUTOPEP8=~/.local/bin/autopep8
```

Comment:
We need a better error message when autopep8 cannot be found on the system, and ideally default to python installation automatically. 
Also this document should focus on vulkan generator development related items only, currently this is a core requirement for the presubmit tests and such information should be under compiling.md or agi/devdoc.md

Reply:
We cannot change the error message as it's given by the bash. 

We can restructure the doc later if we want. But I think we should at least land this document so that other people in the team can run presubmit.

Reply:
I've made some changes for presubmit tests in PR #1199, this one can be closed now.